src/routes/(console)/project-[region]-[project]/auth/teams/+page.svelte

```diff
@@ -19,17 +19,72 @@
     import type { Models } from '@appwrite.io/console';
     import { writable } from 'svelte/store';
     import { canWriteTeams } from '$lib/stores/roles';
-    import { Icon, Layout, Table } from '@appwrite.io/pink-svelte';
+    import {
+        Icon,
+        Layout,
+        Table,
+        FloatingActionBar,
+        Badge,
+        Typography
+    } from '@appwrite.io/pink-svelte';
     import { IconPlus } from '@appwrite.io/pink-icons-svelte';
     import DualTimeView from '$lib/components/dualTimeView.svelte';
+    import { sdk } from '$lib/stores/sdk';
+    import { Submit, trackError, trackEvent } from '$lib/actions/analytics';
+    import { Dependencies } from '$lib/constants';
+    import { addNotification } from '$lib/stores/notifications';
+    import { invalidate } from '$app/navigation';
+    import Confirm from '$lib/components/confirm.svelte';
 
     export let data;
 
     const region = page.params.region;
     const project = page.params.project;
+
+    let selectedTeams: string[] = [];
+    let showDelete = false;
+    let deleting = false;
+
+    const columns = writable([
+        { id: 'name', title: 'Name', type: 'string', width: { min: 200, max: 300 } },
+        { id: 'members', title: 'Members', type: 'string', width: { min: 120, max: 150 } },
+        { id: 'created', title: 'Created', type: 'string', width: { min: 120, max: 180 } }
+    ]);
```

Comment:
we previously used a fixed width column, was this new width change approved by design though?